proto/loopback_tutorial.html

```diff
@@ -36,61 +36,97 @@
 </div>
 
 <div class="downtheheader">
-<table>
-<tr>
-<td>
-    <ul class="nav navbar-nav navbar-left">
-      <li id="gettingstarted">
-        <img src="../images/back.svg" class="back"/> 
-      </li>
-      <li id="gettingstarted">
-      <p class="gs">Getting Started</p>
-      </li>
-      </ul>
-      </td>
-</tr>
-<tr>
-<td>
-      <li id="gettingstarted">
-				 <p class="tutorialtitle">Creating your first microservice</p>
-      </li>
-      </td>
-</tr>
-</table>
-
+  <div class="gettingstarted">
+    <img src="../images/back.svg" class="back"/> 
+    <p class="gs">Getting Started</p>
+  </div>
+	<p class="tutorialtitle">Creating your first microservice</p>
 </div>
+
 <div class="row">
   <div class="col-md-4">
-    <ul id="table">
+    <ul class="table">
     <p  id="tabletitle">Table of Contents</p>
-      <li  id = "contentsselected"><span>Introduction</span></li>
+    <span class='line'></span>
+      <li  id = "contentsselected"><span id = "contentsselected">Introduction</span></li>
 
-      <li id="contents"><span>Create a new project</span></li>
+      <li id="contents"><span id="contents">Create a new project</span></li>
```

Comment:
`id="contents"` can only be used once because its an `id`

please fix in another PR (we can merge this to get the current HTML updated for preview)